perceval/components/unitary_components.py

```diff
@@ -415,6 +415,32 @@ def apply(self, r, sv):
         })
         return nsv
 
+    def break_in_2_mode_perms(self):
+        """
+        Breaks any n-mode PERM into an equivalent circuit with only 2 mode PERMs
+
+        :return: An equivalent Circuit with only 2 mode PERM components
+        """
+
+        perm_vec_req = self.perm_vector
+        perm_len = len(perm_vec_req)
+
+        if perm_len == 2:
+            print("length 2, returning myself")
```

Comment:
Is this a debug trace? Can you remove it, please?

Reply:
Oops, yes I missed to delete it. Thanks for catching it.